.github/workflows/main.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     name: Build
     strategy:
       matrix:
-        rust-version: ["1.58", "stable"]
```

Comment:
The idea is to run for the minimal Parrot supports and for the current stable one.
Have you tried 1.59? If it works, change this to 1.59, if not 1.60, if not 1.61 and if not 1.62

Reply:
Ah, alright. I thought we were just trying to keep up with latest stable release.

Reply:
That's the "stable" portion yeah